src/libs/antares/study/parts/thermal/container.cpp

```diff
@@ -39,12 +39,7 @@ namespace Antares
 namespace Data
 {
 PartThermal::PartThermal() :
- unsuppliedEnergyCost(0.),
- spilledEnergyCost(0.),
- list(),
- mustrunList(),
- clusters(nullptr),
- clusterCount((uint)-1)
+ unsuppliedEnergyCost(0.), spilledEnergyCost(0.), list(), mustrunList(), clusters()
```

Comment:
Same thing as above : 
Do we really need the call to default `clusters()` constructor ?

Reply:
No, I removed this call and others.